src/AcmForm/AcmForm.stories.tsx

```diff
@@ -11,27 +12,56 @@ export default {
 }
 
 export const Form = () => {
-    const [value, setValue] = useState<string>()
+    const [email, setEmail] = useState<string>()
+    const [password, setPassword] = useState<string>()
+    const [description, setDescription] = useState<string>()
     const [select, setSelect] = useState<string>()
     const [labels, setLabels] = useState<string[]>()
     return (
         <AcmForm>
             <AcmTextInput
-                label="Text Input"
                 id="textInput"
-                value={value}
-                onChange={setValue}
+                label="Email"
+                placeholder="Enter email"
+                labelHelpTitle="Email Help"
+                labelHelp="The email must be in valid email format."
+                value={email}
+                onChange={setEmail}
+                validation={(value) => {
+                    if (typeof value === 'string') {
+                        const regExp = /^[\w-]+(\.[\w-]+)*@([a-z0-9-]+(\.[a-z0-9-]+)*?\.[a-z]{2,6}|(\d{1,3}\.){3}\d{1,3})(:\d{4})?$/
+                        if (!regExp.test(value)) return 'Must be a valid email.'
+                    }
+                }}
                 isRequired
-                validation={() => undefined}
             />
             <AcmTextInput
-                label="Secret Input"
                 id="secretInput"
-                value={value}
-                onChange={setValue}
+                label="Password"
+                placeholder="Enter password"
+                labelHelpTitle="Password Help"
+                labelHelp="The password must be at least 8 characters long."
                 type="password"
+                value={password}
+                onChange={setPassword}
+                validation={(value) => {
+                    if (typeof value === 'string') {
+                        if (value.length < 8) return 'Password must be at least 8 characters.'
+                    }
+                }}
                 isRequired
             />
+            <AcmTextArea
+                id="textArea"
+                label="Description"
+                placeholder="Enter description"
+                labelHelpTitle="Help Title"
+                labelHelp="This is the help for the text area"
+                value={description}
+                onChange={setDescription}
+                isRequired
+            />
+
```

Comment:
The TextArea story doesn't need the validation example?

Reply:
The validation is tested in the AcmTestArea.test.tsx.
The form story is just giving an example of a full form.

Reply:
Ah okay, I just saw the other controls were given a validation to show as an example